test-integration/simple-project-truffle-web3-1.2.0/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+
+cd "$(dirname $0)"
+
+cp -r ../simple-project-truffle-template .
+
+sed -i "s/\"truffle\": \"5.0.1\"/\"truffle\": \"5.0.29\"/g" simple-project-truffle-template/package.json
```

Comment:
`-i` with no arguments is not portable, it doesn't work in macOS.

```suggestion
sed -i.bak "s/\"truffle\": \"5.0.1\"/\"truffle\": \"5.0.29\"/g" simple-project-truffle-template/package.json
rm simple-project-truffle-template/package.json.bak
```